api/v1alpha1/biosversion_types.go

```diff
@@ -0,0 +1,119 @@
+// SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and IronCore contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	"github.com/stmcginnis/gofish/common"
+	"github.com/stmcginnis/gofish/redfish"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type BIOSVersionState string
+
+const (
+	// BIOSVersionStatePending specifies that the bios upgrade maintenance is waiting
+	BIOSVersionStatePending BIOSVersionState = "Pending"
+	// BIOSVersionStateInProgress specifies that upgrading bios is in progress.
+	BIOSVersionStateInProgress BIOSVersionState = "Processing"
+	// BIOSVersionStateCompleted specifies that the bios upgrade maintenance has been completed.
+	BIOSVersionStateCompleted BIOSVersionState = "Completed"
+	// BIOSVersionStateFailed specifies that the bios upgrade maintenance has failed.
+	BIOSVersionStateFailed BIOSVersionState = "Failed"
+)
+
+type UpdatePolicy string
+
+const (
+	UpdatePolicyForce  UpdatePolicy = "Force"
+	UpdatePolicyNormal UpdatePolicy = "Normal"
```

Comment:
Btw do we really need `Normal`? I would assume if the `Policy` is not set the reguar flow should be the `Normal` way of applying an update.